### PR TITLE
Add OpenShift Pipelines 4.15 job for HyperShift

### PR DESCRIPTION
**Pull request by liswang89**

### CODE REVIEW

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.14-openshift-pipelines-ocp4.15-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.15-lp-rosa-hypershift.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                {"step": "rosa-cluster-wait*", "failure_type": "all", "classification": "Infrastructure Provisioning - Rosa Cluster Operators", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+                {"step": "rosa-sts-account-roles-create", "failure_type": "all", "classification": "Account Roles Creation - AWS", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 2},"jira_additional_labels": ["!default"]},
+                {"step": "rosa-sts-account-roles-delete", "failure_type": "all", "classification": "Account Roles Deletion - AWS", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 3}, "jira_additional_labels": ["!default"]},
+                {"step": "rosa-conf-idp-htpasswd", "failure_type": "all", "classification": "Admin Access - Cluster", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]}
```
Comment (oharan2) [inline ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.15-lp-rosa-hypershift.yaml:63]:
```suggestion
                {"step": "openshift-pipelines-install", "failure_type": "all", "classification": "Operator Installation Failure", "group": {"name": "lp-tests", "priority": 1},"jira_additional_labels": ["!default"]},
                {"step": "openshift-pipelines-tests", "failure_type": "all", "classification": "Test Failure", "jira_project": "SRVKP", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default"]}
```

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.15-lp-rosa-hypershift.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+              ]
+        }
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.15-lp","rosa-hypershift-lp","pipelines-lp"]'
```
Comment (oharan2) [inline ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.15-lp-rosa-hypershift.yaml:66]:
```suggestion
      FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/rosa/lp-interop.json
      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.15-lp","rosa-hypershift-lp","pipelines-lp"]'
```

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.15-lp-rosa-hypershift.yaml (hunk 1 above)
Comment (shakyav) [reply ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.15-lp-rosa-hypershift.yaml:63]: +1

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.15-lp-rosa-hypershift.yaml (hunk 2 above)
Comment (shakyav) [reply ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.15-lp-rosa-hypershift.yaml:66]: +1

Comment (liswang89) [reply ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.15-lp-rosa-hypershift.yaml:66]: @oharan2 Is the environmental variable FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS necessary? Is it always in the same location?

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.15-lp-rosa-hypershift.yaml (hunk 1 above)
Comment (liswang89) [reply ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.15-lp-rosa-hypershift.yaml:63]: I will resolve it in another PR.

**Pull request merged**